### DISCUSSION TITLE
HTML body width much smaller on iOS chrome.

### DISCUSSION THREAD

**Discussion by BustosAndrew** [saadeghi/daisyui#2402] [Q&A]
What could be the reason why everything appears normal on desktop, and even within instagram's site preview, but not on iOS chrome? It appears fine on iOS Safari, however. See pic below:
![IMG_3344](https://github.com/saadeghi/daisyui/assets/22780026/eddc3da4-8156-4f8e-aefc-ebb3ac6d16e4)
Repo: https://github.com/BustosAndrew/herrera-builders-llc

**Comment by BustosAndrew** [comment]
Ok so the reason why is because I'm including an svg in the button in the nav. Any workaround for this?

**Comment by saadeghi** [comment]
Looks like you already fixed the size issue.
